packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -63,7 +64,24 @@ abstract class ProgressIndicator extends StatefulWidget {
 
   /// The progress indicator's background color.
   ///
-  /// The current theme's [ColorScheme.background] by default.
+  /// This color is interpreted differently depending on the type of
```

Comment:
I would think that in subclasses you could write:
```dart
/// Subclass doc for backgroundColor here
@override
Color get backgroundColor => super.backgroundColor;
```

Reply:
Nice! I didn't think about treating it as a getter override instead of a field override.